code/modules/mob/living/silicon/robot/robot_mob.dm

```diff
@@ -670,33 +670,32 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 	return TRUE
 
 // this function displays the cyborgs current cell charge in the stat panel
-/mob/living/silicon/robot/proc/show_cell_power()
+/mob/living/silicon/robot/proc/show_cell_power(list/status_tab_data)
 	if(cell)
-		stat(null, "Charge Left: [cell.charge]/[cell.maxcharge]")
-	else
-		stat(null, "No Cell Inserted!")
+		return list("Charge Left:", "[cell.charge]/[cell.maxcharge]")
+
```

Comment:
```suggestion

```
not totally sure if this is needed here?

Reply:
Thanks, realised I could simplify it further by making a ternary, appreciated for the pointer